articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+
+
+* Deployment of new markets (silos),
+* Setting/adjusting silo parameters (LTV, LT, interest model, price feed),
+* Minting/redeeming XAI
+* Turning on/off fees
+* Deploying new bridge assets,
+* and even increasing the supply of SILO tokens
+
+In summary, governance has extensive and very far-reaching powers over Silo. Hence, the question arises “how likely is it that a malicious party can obtain a majority voting influence to instigate harmful changes to the protocol” (e.g. mint infinite XAI)?
+
+For this to happen, the party would need to accrue a substantial amount of SILO tokens (or get voting power delegated). Presuming that the team and SILO investors only vote in the best interest of the DAO, the answer is: The possibility is very low. The chart below shows the voting power of all delegators. It’s important to note that to participate in on-chain voting, one has to delegate SILO to one's wallet or another delegate.
+
+
+
+<p id="gdcalert10" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image10.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert11">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image10.png "image_tooltip")
+
+
+(source: [Boardroom - Delegates Voting Power Distribution](https://boardroom.io/silo/delegates))
+
+As seen in the chart, the top three delegates control 51.2% of the overall voting power that is currently eligible to vote. This is quite a high concentration of power (basically a [Nakamoto coefficient](https://news.earn.com/quantifying-decentralization-e39db233c28e) of 3). With a bit of forensics on etherscan, using the [vested token overview](https://docs.google.com/spreadsheets/d/1IcvYQlWQ34kIFIfIKWzJwGlw-Jv33_TMBDKCI0ziEa8/edit#gid=1565912853), it becomes clear that the address with the most delegated voting power is controlled by the founding team. The same is true for ranks 4, 7, and 10. The rest seem to be investors and community members.
+
+This does not come as a surprise, given that the founding team will receive 27% of all tokens in circulation (vested over 3 years) according to the [vesting schedule](https://docs.google.com/spreadsheets/d/1IcvYQlWQ34kIFIfIKWzJwGlw-Jv33_TMBDKCI0ziEa8/edit#gid=1565912853) [side note: the team allocation stated in their [docs](https://silopedia.silo.finance/governance/token-allocation-and-vesting) is 5.25% lower (21.75%). Those tokens are currently sitting in a vesting contract untouched].
+
+In summary, the team has by far the largest allocation. All other stakeholder groups don’t even come close, even all investors combined only achieve 6.3% voting power. While it’s not up to us to judge what token allocation is fair, it still leads to the conclusion that SiloDAO is fully controlled by the founding team, and will be for the foreseeable future.
```

Comment:
Is there a token distribution that plans to shift power from founding team toward protocol users over some time period?

Reply:
yes, they plan veSILO, but no idea when this will come. Added it